packages/flutter/lib/src/widgets/routes.dart

```diff
@@ -108,6 +108,14 @@ abstract class TransitionRoute<T> extends OverlayRoute<T> {
   Future<T?> get completed => _transitionCompleter.future;
   final Completer<T?> _transitionCompleter = Completer<T?>();
 
+  /// Handle to the performance mode request.
+  ///
+  /// When the route is installed, the performance mode is requested, and this is
+  /// then disposed when the route is disposed. Requesting [DartPerformanceMode.latency]
```

Comment:
> disposed when the route is disposed

Really? The entire time the route is shown we want to be in latency mode? I though we only want this for the duration of the transition.